### ISSUE TITLE
[python-dm-tree] fails to build

### ISSUE THREAD

**Issue by moghingold** [acxz/pkgbuilds#245]
This is the output of `makepkg -i` on AUR target python-dm-tree: https://pastebin.com/NUpMyc3t

I get similar results with target python-dm-tree-git. The strange thing is that this package seems to be trying to download and build abseil-cpp, which is available as a separate AUR package (abseil-cpp-git) and had no build issues when I installed it separately.

Is there an upstream issue, or does there seem to be something hinky with my system configuration?

**Comment by moghingold** [comment]
Very similar output for `makepkg -i` on AUR target python-dm-tree-git: https://pastebin.com/GkrhpvEM

**Comment by moghingold** [comment]
Can also now confirm that the problem persists between boots. So if it's an issue with my system state then it isn't a temporary one.

**Comment by moghingold** [comment]
But I installed abseil-cpp and python-dm-tree still tries to build it anyways. It's not detecting and using the installed version. So the build script seems doubly-wrong. Is there an upstream I should take this info to? Or is this addressable through the PKGBUILD?

**Comment by intelfx** [comment]
@moghingold yes, it is trying to build its own abseil-cpp unconditionally. This is addressable through the PKGBUILD by de-vendoring abseil, at the cost of making this package dependent on system abseil-cpp. 

Alternatively, the build system can be modified to vendor abseil-cpp _correctly_, but I am uninterested in doing that :)

**Comment by acxz** [comment]
Keeping this open until the patch is not needed.

**Comment by acxz** [comment]
patched with #247

**Comment by xangelix** [comment]
please bump pkgrel in the future to make sure people actually get the fix

**Comment by jaw-sh** [comment]
I am unfamiliar with the Python development environment, but I am unable to install the Arch package for python-dm-tree(-git) due to the following build error.
[python-dm-tree.txt](https://github.com/user-attachments/files/17085568/python-dm-tree.txt)
